storage/innobase/log/log0online.cc

```diff
@@ -290,7 +290,7 @@ log_online_read_bitmap_page(
 	     <= bitmap_file->size - MODIFIED_PAGE_BLOCK_SIZE);
 	ut_a(bitmap_file->offset % MODIFIED_PAGE_BLOCK_SIZE == 0);
 
-	IORequest io_request(IORequest::LOG | IORequest::READ);
+	IORequest io_request(IORequest::LOG | IORequest::READ | IORequest::NO_ENCRYPTION);
```

Comment:
This works, but that still keeps the IORequest::LOG in mix, which is not entirely correct. Have you evaluated effort needed to move away from that?

Reply:
I did - and that would require changing multiple conditions in os0file, and adding a new enum value. The `NO_ENCRYPTION` tag is already used in a similar case for keyring encryption, so it seemed a better choice.